src/Collections-Tests/ArrayTest.class.st

```diff
@@ -753,9 +753,9 @@ ArrayTest >> testPrinting [
 ArrayTest >> testShuffleBy [
 
 	| base random shuffleByResult |
-	base := #(1 2 3 4 5).
+	base := #(1 2 3 4 5) copy.
```

Comment:
why do we need the two copy?
If shuffled by change base I understand the copy but I do not get the need for the result.

Reply:
if we hand over a literal array, it will be modified. that means, after running the test, the literal is not in sync with the source. (bugs like that are impossible to find...)

If we hand over the copy, only the copy will be modified.

Alternative are: use Array with:with:... or {}.